src/views/WordBazaarAssets.vue

```diff
@@ -1,33 +1,87 @@
 <template>
   <div class="word-bazaar-assets">
-    <TabBar
-      v-model="activeTab"
-      :tabs="tabs"
-    >
-      <SearchInput
-        v-if="showSearch"
-        v-model="search"
-        :placeholder="$t('views.WordBazaar.Placeholder')"
-        class="desktop"
-        @input="reloadData"
-        @close="showSearch = false"
-      />
-      <IconSearch
-        v-else
-        @click="showSearch = true"
-      />
+    <TabBar class="desktop">
+      <template slot="left">
+        <TabBarButton
+          :class="{ active: activeTab === 'all' }"
+          @click="activeTab = 'all'"
+        >
+          {{ $t('views.WordBazaar.Tabs[0].text') }}
```

Comment:
```suggestion
          <span class="desktop">{{ $t('views.WordBazaar.Tabs[0].text') }}</span>
          <span class="mobile">{{ $t('views.WordBazaar.Tabs[0].textMobile') }}</span>
```
As we agreed, we won't be using different wording in mobile and desktop versions. But for now, I'm proposing switch wording this way, to don't duplicate a lot of markup.